### PR TITLE
cleanup of food_and_drink/plants.dm

### PR DESCRIPTION
**Pull request by NibChocolateeny**
[C-Code-Quality][A-Hydroponics]

## About the PR
cleans up modules/food_and_drink/plants.dm and makes code/descriptions nicer. this means:
- make "edible" var all true/false instead of ture/false and 1/0
- break up some long comments and lines for read ability (not all - not sure if it will break things)
- gets rid of broken description on bean plantes (I can fix in later pr!)
- minor grammar/wording fixes to descriptions aned other messages (onions red, not yellow; coconut not cut in slice; etc)

## Why's this needed? 
better code! Maybe? reviewers tell me, maybe it is not so good :<

### CODE REVIEW

**Comment by NibChocolateeny** [comment]
...why are the tags not there?

**Comment by pgmzeta** [comment]
> ...why are the tags not there?

it doesn't expect you to put the A/C as a prefix on your tags (e.g. `[Hydroponics]` would work).

File: code/modules/food_and_drink/plants.dm [hunk 1]
```diff
@@ -488,11 +491,12 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 /obj/item/reagent_containers/food/snacks/plant/melon/george
 	name = "rainbow melon"
 	crop_prefix = "rainbow "
-	desc = "Sometime in the year 2472 these melons were required to have their name legally changed to protect the not-so-innocent. Also for tax evasion reasons."
+	desc = "Sometime in the year 2472 these melons were required to have their name legally changed to protect the not-so-innocent. \
```
Comment (TheTyrant25) [inline code/modules/food_and_drink/plants.dm:494]: i know it wasnt you who wrote this but the game is set in the year 2053. 2472 is... quite a lot after that. perhaps change the year.

File: code/modules/food_and_drink/plants.dm [hunk 2]
```diff
@@ -777,7 +783,7 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 
 /obj/item/reagent_containers/food/snacks/plant/pear
 	name = "pear"
-	desc = "Whether or not you like the taste, its freshness is appearant."
+	desc = "Whether or not you like the taste, its freshness is appearant." // boo!
```
Comment (TheTyrant25) [inline code/modules/food_and_drink/plants.dm:786]: ?

**Review by TheTyrant25 — APPROVED** [review]
Looks mostly alright. You definitely double checked to make sure there's no weird instances of `edible == 2`, right?

File: code/modules/food_and_drink/plants.dm (hunk 1 above)
Comment (NibChocolateeny) [reply code/modules/food_and_drink/plants.dm:494]: ok!

**Comment by NibChocolateeny** [comment]
@TheTyrant25 I lookd at rest of code with "edible," saw only mix of 1/0/TRUE/FALSE, no 2 or -1. maybe next fix is to standardize eddible in other files?

File: code/modules/food_and_drink/plants.dm [hunk 3]
```diff
@@ -488,11 +491,14 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 /obj/item/reagent_containers/food/snacks/plant/melon/george
 	name = "rainbow melon"
 	crop_prefix = "rainbow "
-	desc = "Sometime in the year 2472 these melons were required to have their name legally changed to protect the not-so-innocent. Also for tax evasion reasons."
+	desc = "Back in 2013, these melons were required to have their name legally changed to protect the not-so-innocent. \
```
Comment (ZeWaka) [inline code/modules/food_and_drink/plants.dm:494]:
```suggestion
	desc = "Back in 2013, these melons were required to have their name legally changed to protect the not-so-innocent. \
```
we don't reference real life in the game generally, please change this

File: code/modules/food_and_drink/plants.dm [hunk 4]
```diff
@@ -1522,7 +1533,7 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 
 /obj/item/reagent_containers/food/snacks/plant/purplegoop
 	name = "purple goop"
-	desc = "Um... okay then."
+	desc = "Um...okay then."
```
Comment (ZeWaka) [inline code/modules/food_and_drink/plants.dm:1536]: why

File: code/modules/food_and_drink/plants.dm [hunk 5]
```diff
@@ -632,7 +639,7 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 /obj/item/reagent_containers/food/snacks/plant/chili
 	name = "chili pepper"
 	crop_suffix = " pepper"
-	desc = "Caution: May or may not be red hot."
```
Comment (ZeWaka) [inline code/modules/food_and_drink/plants.dm:635]: i disagree

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**